packages/apps/dev-wallet/src/pages/select-profile/select-profile.css.ts

```diff
@@ -23,19 +46,19 @@ export const imgClass = style([
     alignItems: 'center',
     overflow: 'hidden',
     borderRadius: 'xxl',
-    marginBlockEnd: 'md',
   }),
   {
-    width: '100px',
-    height: '100px',
-    border: `2px solid ${borderColor}`,
+    width: '40px',
+    height: '40px',
+    borderRadius: '2px',
```

Comment:
this should be removed as there is already `borderRadius: 'xxl',` specified above and based on the design it's should be rounded